common/src/main/java/com/microsoft/identity/common/internal/telemetry/events/HttpStartEvent.java

```diff
@@ -35,23 +37,28 @@ public HttpStartEvent putMethod(String method) {
         return this;
     }
 
-    public HttpStartEvent putPath(String path) {
-        put(Key.HTTP_PATH, path);
-        return this;
-    }
+    public HttpStartEvent putPath(URL path) {
+        final StringBuilder logPath = new StringBuilder();
+        logPath.append(path.getProtocol())
+                .append("://")
+                .append(path.getAuthority())
+                .append('/');
 
-    public HttpStartEvent putRequestIdHeader(String requestIdHeader) {
-        put(Key.HTTP_REQUEST_ID_HEADER, requestIdHeader);
-        return this;
-    }
+        // we do not want to send tenant information
+        // index 0 is blank
+        // index 1 is tenant
+        final String[] splitArray = path.getPath().split("/");
+        for (int i = 2; i < splitArray.length; i++) {
```

Comment:
This looks like a potential hazard for b2c urls where the format is pretty undefined (now that vanity urls are supported) - we may need to take a // TODO for unit testing here.

Reply:
We could also cross-reference w/ iOS to see what they're doing - we could just flag the whole UJRL as PII/OII and not do any magic on it at all

Reply:
iOS is using the absoluteString  of the url. https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/68aa3f4116c818fb4486a56d067641858ee575d5/IdentityCore/src/telemetry/MSIDTelemetryAuthorityValidationEvent.m#L40

Reply:
Ah - so looks like they're doing nothing fancy at all. That's probably what we should do.

Reply:
The reason is the same with [ADAL telemetry](https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/01a117bf3f544d527d4f41a99c5dcb79bc745163/adal/src/telemetry/java/com/microsoft/aad/adal/HttpEvent.java#L64) where we want to filter out the query parameters as well as the tenant info from the path.